solution/ui/e2e/cypress/fixtures/42.433.10.public-formatted.js

```diff
@@ -216,7 +216,7 @@ export default [
         description: "SMDLs, SHOs, CIBs, FAQs, SMM",
         id: 4,
         is_fr_link_category: false,
-        name: "Subregulatory Test Guidance",
+        name: "Subregulatory Guidance",
```

Comment:
This was too clever by half and was screwing up the homepage tests.  We need this to be Subregulatory Guidance in order to fetch the data from this fixture for the Recent Guidance tab